jaeger_client/constants.py

```diff
@@ -33,10 +33,10 @@
 DEFAULT_FLUSH_INTERVAL = 1
 
 # Name of the HTTP header used to encode trace ID
-TRACE_ID_HEADER = b'uber-trace-id'
```

Comment:
@yurishkuro Seems like I can't reproduce the error, so a test case would be beneficial...

I'm also not sure how switching `'uber-trace-id'` to `b'uber-test-case'` would help alone – to my knowledge these two should be equivalent in Python 2.

Here is what I've tried to do (with some variations):
```
# coding=utf-8

import urllib2

headers = {
    'User-Agent': 'Mozilla/5.0',
    'header_хидер': (u'value_значение').encode('utf-8'),
    'header_klüft_skräms_große': (u'À quelle fréquence envoyez-vous des données étranges?').encode('utf-8')
}

body = 'uber-trace-id'

r = urllib2.Request('http://localhost:11111', data=body, headers=headers)
response = urllib2.urlopen(r)
data = response.read()

print(response)
print('\n\n----------------------\n\n')
print(data)
```